frontend/mono-ui/web/rainmaker/dev-packages/egov-bnd-dev/src/ui-config/screens/specs/utils/index.js

```
@@ -758,7 +758,7 @@ export const postPaymentActivity = async (
         { key: "tenantId", value: data.tenantId },
         { key: "consumerCode", value: data.consumerCode },
       ];
-      let module = data.businessService == "BIRTH_CERT" ? "birth" : "death";
+      let module = data.businessService == "BIRTH_CERT.BIRTH_CERT" ? "birth" : "death";
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'module' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)